pkg/transformers/image.go

```diff
@@ -88,38 +87,52 @@ func (pt *imageTransformer) findAndReplaceImage(obj map[string]interface{}) erro
 	return nil
 }
 
-func (pt *imageTransformer) updateContainers(in interface{}) (interface{}, error) {
-	containers, ok := in.([]interface{})
-	if !ok {
-		return nil, fmt.Errorf("containers path is not of type []interface{} but %T", in)
+func (pt *imageTransformer) updateContainer(container map[string]interface{}) error {
```

Comment:
I suggest to add another function `updateImage` which updates one image. And the mutate function will call `updateImage` instead of `updateContainers`.  Then you don't need to change the logic in `updateContainers`.

Reply:
done